<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Courses</class>
 <widget class="QDialog" name="Courses">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1154</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #3a3f5c;</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1154</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>LBTC SMART RECORD SYSTEM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Course_back">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>50</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    
QPushButton{
                    color: rgb(255, 255, 255);
                    border:2px solid green;
                    border-radius: 8px;
                    padding: 0 8px;
                    background-color: rgb(255, 267, 245);
                    font: 500 11pt ;
                   

}

 </string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::GoHome"/>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>971</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLineEdit" name="lineEdit_Department">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">   
QLineEdit{
                    color: rgb(255, 255, 255);
                    border:1px solid green;
                    border-radius: 0px;
                    padding: 0 8px;
                    background-color: #3a3f5c;
                    
	                font: 700 9pt &quot;Segoe UI&quot;;
                   

}

 </string>
    </property>
    <property name="placeholderText">
     <string>Department</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_code">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">   
QLineEdit{
                    color: rgb(255, 255, 255);
                    border:1px solid green;
                    border-radius: 0px;
                    padding: 0 8px;
                    background-color: #3a3f5c;
                    
	                font: 700 9pt &quot;Segoe UI&quot;;
                   

}

 </string>
    </property>
    <property name="placeholderText">
     <string>CODE</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Course_Name">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">   
QLineEdit{
                    color: rgb(255, 255, 255);
                    border:1px solid green;
                    border-radius: 0px;
                    padding: 0 8px;
                    background-color: #3a3f5c;
                    
	                font: 700 9pt &quot;Segoe UI&quot;;
                   

}

 </string>
    </property>
    <property name="placeholderText">
     <string>Program</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Total_Subjects">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>110</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">   
QLineEdit{
                    color: rgb(255, 255, 255);
                    border:1px solid green;
                    border-radius: 0px;
                    padding: 0 8px;
                    background-color: #3a3f5c;
                    
	                font: 700 9pt &quot;Segoe UI&quot;;
                   

}

 </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Total Subjects</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Course_Duration">
    <property name="geometry">
     <rect>
      <x>248</x>
      <y>173</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">   
QLineEdit{
                    color: rgb(255, 255, 255);
                    border:1px solid green;
                    border-radius: 0px;
                    padding: 0 8px;
                    background-color: #3a3f5c;
                    
	                font: 700 9pt &quot;Segoe UI&quot;;
                   

}

 </string>
    </property>
    <property name="placeholderText">
     <string>Course Duration</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Course_Submit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>350</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">    
QPushButton{
                    color: rgb(255, 255, 255);
                    border:2px solid green;
                    border-radius: 8px;
                    padding: 0 8px;
                    background-color: rgb(255, 267, 245);
                    font: 500 11pt ;
                   

}

 </string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::DocumentSend"/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>971</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">    
QPushButton{
                    color: rgb(255, 255, 255);
                    border:1px solid green;
                    border-radius: 6px;
                    padding: 0 8px;
                    background-color: rgb(255, 267, 245);
                    font: 500 11pt ;
                   

}

 </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Reset_Course">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>350</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">   
QPushButton{
                    color: rgb(255, 255, 255);
                    border:2px solid green;
                    border-radius: 8px;
                    padding: 0 8px;
                    background-color: rgb(255, 267, 245);
                    font: 400 11pt ;
                   

}

 </string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::ViewRestore"/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Department</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>497</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>CODE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>497</x>
      <y>80</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Subjects</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>80</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Course Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Course Duration</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_SubjectList">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>170</y>
      <width>321</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">   
QPlainTextEdit{
                    color: rgb(255, 255, 255);
                    border:1px solid green;
                    border-radius: 0px;
                    padding: 0 8px;
                    background-color: #3a3f5c;
                    
	                font: 700 9pt &quot;Segoe UI&quot;;
                   

}

 </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>150</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Subject</string>
    </property>
   </widget>
   <zorder>btn_Course_Submit</zorder>
   <zorder>lineEdit_Department</zorder>
   <zorder>lineEdit_code</zorder>
   <zorder>lineEdit_Course_Name</zorder>
   <zorder>lineEdit_Total_Subjects</zorder>
   <zorder>lineEdit_Course_Duration</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>btn_Reset_Course</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>plainTextEdit_SubjectList</zorder>
   <zorder>label_8</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
